modules/.vale.ini

```diff
@@ -14,6 +14,8 @@ AsciiDocDITA.BlockTitle = error
 AsciiDocDITA.CalloutList = error
```

Comment:
Noted in Slack, but this is missing AssemblyContents. They should be identical lists between the two, even if not all rules apply to both modules and assemblies.